website/blog/2022-04-19-dbt-cloud-postman-collection.md

```diff
@@ -0,0 +1,179 @@
+
+The main goal of this article is to spread awareness of these endpoints as the docs are being built & show you how to use them. 
+
+<!--truncate-->
+
+You can use this blog post as an entry point to this [postman collection](https://documenter.getpostman.com/view/14183654/UVsSNiXC#auth-info-7ad22661-8807-406b-aec0-34a46b671aac) to help you automate previously manual tasks, such as managing dbt Cloud account infrastructure, creating dbt Cloud projects, database connections, and managing users.
+
+Please keep in mind that the collection is not an evergreen doc. We are actively developing and updating the API endpoints used to interact with dbt Cloud, so endpoint urls and / or expected request formats may change. Each endpoint in the collection provides comment capabilities, so please notify us there if something looks amiss.
+
+With all of that being said, why should dbt users care about automating dbt Cloud account management?
+
+## What problems does programmatic dbt Cloud account management solve?
+
+When first learning dbt Cloud and operating at a smaller scale, most end users tend to favor manual, GUI based workflows for creating and managing their account infrastructure. Many of our customers start off this way, but they usually end up running into these bottlenecks as the number of jobs and projects multiplies. Here are some examples of the kind of bottlenecks I hear our growing customers talk about.
+
+- “I find myself wasting time manually clicking through the UI to manage infrastructure, especially after deploying more than a handful of projects and environments.”
+- “I don’t have peace of mind that my production environment is configured exactly as intended. If a new team member deletes or changes something, I have to run a fire drill to fix it.”
+- “My organization requires me to run any environmental configuration changes through code review and version control.”
+
+We usually advise our customers to leverage API requests to automate these typically manual tasks, and you should, too!
+
+## Closing Thoughts
+
+Beyond the day-to-day process of managing their dbt Cloud accounts, many organizations benefit from being able to quickly replicate environments as they experiment with new dbt features and development patterns. Governance is another benefit of managing infrastructure in code, as resource definitions can be version controlled and inspected by the relevant teams before changes go live. These concepts are high on our minds as we look to provide additional capabilities to our customers in their use of dbt Cloud. Happy APIing!
+
+*Below this you’ll find a series of example requests - use these to guide you or [check out the Postman Collection](https://dbtlabs.postman.co/workspace/Team-Workspace~520c7ac4-3895-4779-8bc3-9a11b5287c1c/request/12491709-23cd2368-aa58-4c9a-8f2d-e8d56abb6b1dlinklink) to try it out yourself.*
+
+## Examples of how to use the Postman Collection
```

Comment:
Should we add 'Appendix:' here - feels kind of odd to have closing thoughts then another super long section - think the appendix callout would clear it up